sdk/python/lib/test/test_next_serialize.py

```diff
@@ -1020,3 +1021,39 @@ def transformer(prop: str) -> str:
                 "foo_baz": "world",
             },
         }, prop)
+
+
+class StrEnum(str, Enum):
+    ONE = "one"
+    ZERO = "zero"
+
+
+class IntEnum(int, Enum):
+    ONE = 1
+    ZERO = 0
+
+
+class FloatEnum(float, Enum):
+    ONE = 1.0
+    ZERO_POINT_ONE = 0.1
+
+
+class EnumSerializationTests(unittest.TestCase):
+    @async_test
+    async def test_string_enum(self):
+        one = StrEnum.ONE
+        prop = await rpc.serialize_property(one, [])
+        self.assertEqual(StrEnum.ONE, prop)
+
+    @async_test
+    async def test_int_enum(self):
+        one = IntEnum.ONE
+        prop = await rpc.serialize_property(one, [])
+        self.assertEqual(IntEnum.ONE, prop)
+
+    @async_test
+    async def test_float_enum(self):
+        one = FloatEnum.ZERO_POINT_ONE
+        prop = await rpc.serialize_property(one, [])
+        self.assertEqual(FloatEnum.ZERO_POINT_ONE, prop)
+
```

Comment:
In a somewhat surprising turn of events, it turns out that these serialize just fine.